lib/matplotlib/widgets.py

```diff
@@ -2457,22 +2440,14 @@ def onselect(verts):
 
     Parameters
     ----------
-    ax : :class:`~matplotlib.axes.Axes`
+    ax : `~matplotlib.axes.Axes`
         The parent axes for the widget.
     onselect : function
         Whenever the lasso is released, the *onselect* function is called and
         passed the vertices of the selected path.
-    button : List[Int], optional
-        A list of integers indicating which mouse buttons should be used for
-        rectangle selection. You can also specify a single integer if only a
-        single button is desired.  Default is ``None``, which does not limit
-        which button can be used.
-
-        Note, typically:
-
-        - 1 = left mouse button
-        - 2 = center mouse button (scroll wheel)
-        - 3 = right mouse button
+    button : `.MouseButton` or List[`.MouseButton`], optional
```

Comment:
```suggestion
    button : `.MouseButton` or list of `.MouseButton`, optional
```

according to current numpydoc conventions.

Reply:
fixed